CONTRIBUTING.md

```diff
@@ -392,6 +392,7 @@ Christopher Bruns  |@cmbruns-hhmi |Molmodel
 Randy Radmer       |              |Molmodel
 Charles Schwieters |              |Provided initial multibody code 
 Abhinandan Jain    |              |Underlying spatial algebra formulation
+Thomas Lau         |@thomasklau   |Force Parallelization
```

Comment:
I like to bias the order of this list by recent activity, so I would put your name between Carmichael and Ian Stavness.